files/assets/js/userpage_v.js

```diff
@@ -76,7 +76,10 @@ function transferBux(mobile=false) {
 
 function submitFormAjax(e) {
 	for (elementId in ['message', 'message-mobile', 'message-preview', 'message-preview-mobile']) {
-		document.getElementById(elementId).classList.add("d-none");
+		element = document.getElementById(elementId);
+		if (element != null) {
+			document.getElementById(elementId).classList.add("d-none");
```

Comment:
use `element` instead of getting it again

Reply:
Oops, had to deal with kids when I was about to finish that tweak, then failed.

also I am bad at javascript

One minute :V